wagit/sources/WAGit.cpp

```diff
@@ -0,0 +1,33 @@
+//
+// Created by Ranveer Aggarwal on 27/08/20.
+//
+
+#include "../headers/WAGit.h"
+
+namespace WAGit
+{
+    Repository CreateRepository(const Path& path)
```

Comment:
Shouldn't we be creating the config in this function?

Reply:
We will. // ToDo 😛